.github/workflows/test.yml

```diff
@@ -196,6 +196,71 @@ jobs:
           files: ./packages/client/src/__tests__/coverage/clover.xml
           flags: client,${{ matrix.os }},${{ matrix.queryEngine }}
           name: client-${{ matrix.os }}-${{ matrix.queryEngine }}
+
+  #
+  # CLIENT (functional tests with mini-proxy)
+  #
+  client-dataproxy:
```

Comment:
I think the Data Proxy should be exclusively tested with the binary engine, I think this is what it is deployed with?

Reply:
Yes, that's basically the reason I didn't include it in the Buildkite pipeline, to avoid running the Data Proxy tests twice.

(Although even if we did run it with `PRISMA_CLIENT_ENGINE_TYPE=library`, it wouldn't have any effect, mini-proxy always uses the binary engine)

Reply:
> always uses the binary engine

Oh, I see. Perfect. Thanks!